README.md

```diff
@@ -116,3 +116,57 @@ Use the [execution-role.yaml](examples/execution-role.yaml) file to generate an
 
 Alternatively, you can set the trust relationships of your role via AWS Console: in the IAM dashboard, select your role and click **Trust Relationships**:
 ![Screenshot 2023-03-31 at 17 32 55](https://user-images.githubusercontent.com/5663078/230436500-fb4ee057-b70e-4580-a94d-f56191728117.png)
+
+## Error: 404 (request "INVALID_GROUP_ID") An invalid group ID 6458c1bacde92a3069d69603|default was specified
```

Comment:
FYI - this format is going to be extended in the SDKv2 (we provide more details about errors (SDK method etc.)